PWGHF/D2H/Tasks/taskLc.cxx

```diff
@@ -305,12 +363,43 @@ struct HfTaskLc {
       registry.fill(HIST("Data/hImpParErrProng1"), candidate.errorImpactParameter1(), pt);
       registry.fill(HIST("Data/hImpParErrProng2"), candidate.errorImpactParameter2(), pt);
       registry.fill(HIST("Data/hDecLenErr"), candidate.errorDecayLength(), pt);
+
+      if constexpr (applyMl) {
+        double Mass(-1);
+        double outputBkg(-1);
+        double outputFD(-1);
+        if (candidate.isSelLcToPKPi() >= selectionFlagLc) {
+          Mass = hfHelper.invMassLcToPKPi(candidate);
+          outputBkg = candidate.mlProbLcToPKPi()[0]; /// bkg score
+          outputFD = candidate.mlProbLcToPKPi()[1];  /// non-prompt score
```

Comment:
Typically we put the FD score in 3rd position (this is not mandatory, but it is what we usually do, so it would be good to have the same convention everywhere in the code)

Moreover, be careful that depending on the value of `selectionFlagLc` the vectors with BDT scores might be empty. You should add a protection (see e.g. https://github.com/AliceO2Group/O2Physics/blob/063c3880030ab07b026d4bf711ab10bdeef5caa7/PWGHF/D2H/Tasks/taskCharmPolarisation.cxx#L227-L251)

Reply:
yes, you are right! It's a mistake! Thanks for spotting this!